core/common/src/main/java/alluxio/exception/PreconditionMessage.java

```diff
@@ -74,29 +70,14 @@
   // SEMICOLON! minimize merge conflicts by putting it on its own line
   ;
 
-  private final MessageFormat mMessage;
+  private final String mMessage;
 
   PreconditionMessage(String message) {
```

Comment:
This class is pretty similar to `Constants`, where we use `public static final String`, so I think option 2 makes sense from a consistency and efficiency perspective. Option 3 is interesting, but seems like an abuse of `interface` and is more confusing/nonstandard than just `public static final String` everywhere. My order of preference is 2, 3, 1